lever.lua

```diff
@@ -0,0 +1,209 @@
+-- Inspect and pull levers
+world = df.global.world
+local argparse = require('argparse')
+local utils = require('utils')
+
+function leverPullJob(lever, priority)
+    local ref = df.general_ref_building_holderst:new()
+    ref.building_id = lever.id
+
+    local job = df.job:new()
+    job.job_type = df.job_type.PullLever
+    job.pos = {
+        x = lever.centerx,
+        y = lever.centery,
+        z = z
```

Comment:
I don't see where `z` is defined. should this be `lever.z`?

Reply:
ah whoops